cmd/deploy/start_test.go

```diff
@@ -31,7 +31,10 @@ type DeployStartTestSuite struct {
 }
 
 func (suite *DeployStartTestSuite) SetupSuite() {
-	os.Setenv("ARMORY_CLI_TEST", "true")
+	err := os.Setenv("ARMORY_CLI_TEST", "true")
```

Comment:
```suggestion
	suite.NoError(os.Setenv("ARMORY_CLI_TEST", "true"))
```
as written the error is ignored 😦